stylesheets/_component.forms.scss

```diff
@@ -53,6 +47,75 @@ fieldset {
         padding-bottom: 11px;
     }
 
+    &.select {
+        border: 1px solid $input-border;
+        padding: 0 0 0 $padding-base-horizontal;
+
+        @include ie-lte(8) {
+            height: 24px;
+        }
+    }
+
+    &.file {
+        background: none;
+        border: 0;
+        height: auto;
+        line-height: normal;
+        padding: 0;
+
+        @include respond-min($screen-tablet) {
+            padding: 11px 0 0;
+        }
+
+        @include ie-lte(9) {
+            border: 1px solid $input-border;
+            border-radius: 0;
+        }
+    }
+
+    // IE8 - position text in the middle of inputs
+    @at-root {
+        input[type="text"]#{&},
+        input[type="url"]#{&},
+        input[type="password"]#{&},
+        input[type="email"]#{&} {
```

Comment:
http://html5doctor.com/html5-forms-input-types/ - no biggie right now but there's a few types missing that might bring bugs later